ecosystem/typescript/sdk/src/aptos_account.test.ts

```diff
@@ -34,7 +34,7 @@ test("generates derive path accounts", () => {
 
 test("accepts custom address", () => {
   const address = "0x777";
-  const a1 = new AptosAccount(null, address);
+  const a1 = new AptosAccount(undefined, address);
```

Comment:
might be better to have a static method than pass in an undefined...

Reply:
This almost never happens under real use cases. Such account instances cannot sign stuff. `undefined` got passed in here just for simplicity in the test case. We should have passed in some private key bytes here. I don't want to add a static method here. It creates confusion.

Reply:
if in real world case this should never be undefined I think the constructor signature should also not allow passing in undefined. If it's just required for testing to pass in undefined it you can pass in `undefined as any` into the constructor.

Reply:
In a real-world case, it should. `undefined` means or if you pass in nothing, the SDK would create the signing key for you. However, if you intend to pass in address, that means (the auth key has been rotated), you should pass in both.